src/ansys/aedt/core/visualization/post/fields_calculator.py

```diff
@@ -789,7 +789,7 @@ def export(
             )
         else:
             self.__app.logger.error(
-                "You have to provide one of the three following inputs: a path to a file containing the grid of points, "
+                "You have to provide one of the three following inputs: a path to a file containing the grid of points,"
```

Comment:
If you don't add the space back, you'll have ".... points,a sample ..." as error message.

Reply:
fixed by adding a white space at the beginning of the next line.